olp-cpp-sdk-core/include/olp/core/cache/CacheSettings.h

```diff
@@ -77,6 +85,11 @@ struct CacheSettings {
    * @brief Set the disk cache open options.
    */
   OpenOptions openOptions = OpenOptions::Default;
+
+  /**
+   * @brief Path to the readonly cache.
```

Comment:
Would it make sense to explain why the option is called blah_protected and not blah_readonly? Is the meaning of the word "protected" clear from some other place in the docs?